deployment/scripts/cluster.backup.sh

```diff
@@ -40,13 +33,6 @@ kubectl --kubeconfig=${KUBECONFIG} cp \
 kubectl --kubeconfig=${KUBECONFIG} cp \
     default/$(kubectl --kubeconfig=${KUBECONFIG} -n default get pods | grep ocelot-backend |awk '{ print $1 }'):/app/public/uploads $BACKUP_FOLDER/public-uploads
 
-# restart database
-kubectl --kubeconfig=${KUBECONFIG} get deployment ocelot-neo4j -o json \
-    | jq 'del(.spec.template.spec.containers[].command)' \
-    | kubectl --kubeconfig=${KUBECONFIG} apply -f -
-
-# wait for the container to restart
-sleep 60
-
+# set Neo4j in online mode
 # maintenance mode off
-${SCRIPT_DIR}/cluster.maintenance.sh off
\ No newline at end of file
+${SCRIPT_DIR}/cluster.neo4j.sh online-mode
```

Comment:
???

Reply:
Do we not use the maintenance mode here anymore - we have to do this manually?

Reply:
The cluster maintenance mode is included in the database maintenance mode, because it makes no sence to put the database in maintenance without putting the cluster in maintenance. Or?

This way we can use just one command in terminal to put the database in maintenance.
This is cool I think. @ulfgebhardt